### DISCUSSION TITLE
The procedure entry point error on Windows

### DISCUSSION THREAD

**Discussion by WorldOfStories** [jgraph/drawio#3533] [Q&A]
After the last update there was an error on startup (ver 21.2.1)
DiscardVirtualMemory
DLL KERNEL32.DLL

Windows7

**Comment by tupti** [comment]
As far as I checked, the same goes for all systems listed below:

Windows 7 x64
Windows 7 x86
Windows 8.1 x64
Windows 8.1 x86
Windows Server 2008 R2
Windows Server 2012 R2

Are those OSes dropped from support since 21.2.1 release?
I cannot find any information about system requirements for Windows.

**Comment by davidjgraph** [comment]
Yes, 21.2.1 uses electron 24 which does not support Windows 7 or 8, https://www.electronjs.org/blog/windows-7-to-8-1-deprecation-notice . We have to update electron to keep security up to date, so we automatically follow such policies.
